Magento2/Tests/Legacy/WidgetXMLUnitTest.2.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@jcuerdo there has been a misunderstanding, I don't think we should check for bad XML files, what I was talking about was unformatted one like `Magento2/Tests/Legacy/ModuleXMLUnitTest.2.xml`. Removing this file will also allow us to remove the `try...catch` block from the sniff

Reply:
Done ¡¡